### ISSUE TITLE
Settable point-of-interest per player

### ISSUE THREAD

**Issue by Klaranth** [pandorabox-io/in-game#197]
BuckarooBanzai ; Hedgehog ; SX ; wsor 
Reality-check needed on this idea:

Settable point-of-interest per player:
* /visit_set sets the players global visitor-place/entrance
* /visit SomeDude teleports the current player to the previously set destination of the other player (if available)
Use-cases:
* Show others your place even when you are offline, they just have to enter /visit <name>
* Special "players" edge case: /visit expo or /visit Heaven  (or /visit Hell :slight_smile: )
There should also be a privilege guarding this (xp-limited i think) so mass-account creation for abuse won't be a thing

The specials sounds handy. Can /visit_set be set by everyone, or just the one the area is protected by ?

Random position (one of three choices at Fruitland SwissalpS's place) /visit SuicideBooth :p

perhaps players should be /visit player pname as to avoid naming conflicts with /visit place
alternatively you could create a different command to separate ^
or allow players to set locations that are visited with seperate command such as /command pname place
.
course this makes it more complicated

**Comment by Klaranth** [comment]
<shadow@IRC> "visit" word has other contexts. Solid idea though. It's just an access for other players' sethome. I think it might be hilariously over-powered though.
I would like to see a "sethome" for player factions however
admin or staff could add players to a new faction to achieve what you want to do. There's a little formspec plumbing that would need to be done then.
something like '/faction tp' '/faction tphr'

**Comment by S-S-X** [comment]
I think travelhubs should already offer good enough way to do this, also if you do look at current travelhubs you could expect to get at least that many "Interest points". Saying "at least" because setting interest point is simpler, does not consume node and provides free teleporting around.

I think this is good idea if proper restrictions can be applied but very bad idea if implemented as simple command to allow setting any public teleport points. How good it would be for actuaö game play depends heavily on how it would actually be implemented.

* Simple tp public location list wont be good because it would grow out of control for sure.
* XP restriction wont be good because it would make game very unfair.
* Restricting to only visitors wont be good and probably simply causes more alt accounts just to manage teleport point (I would make one right away).
* Restricting amount of locations to very small number wont be good (see point above about alt accounts to manage tp points).
* Removing or disabling interest points if player is not active for x time units is not good way to handle above points.

What else should be taken into account and suggestions how to solve possible problems?

**Comment by SwissalpS** [comment]
I agree that travelhubs already largely cover this. Users can distribute compasses, mission wands etc. in mailboxes and other inventories.
Compass checks for target location's safety, which a /visit command would have to do too.
Players can set location of a full stack of compasses in one click btw.

**Comment by OgelGames** [comment]
Between travelhubs, /tpr and compasses, almost all the use cases are covered. Better to keep those in use rather than adding something easier.
